pandas-stubs/core/groupby/generic.pyi

```diff
@@ -210,38 +210,15 @@ class DataFrameGroupBy(GroupBy, Generic[ByT]):
     def cumsum(self, axis: Axis = ..., **kwargs) -> DataFrame: ...
     def describe(self, **kwargs) -> DataFrame: ...
     def ffill(self, limit: int | None = ...) -> DataFrame: ...
-    @overload
-    def fillna(
-        self,
-        value,
-        method: str | None = ...,
-        axis: Axis = ...,
-        limit: int | None = ...,
-        downcast: dict | None = ...,
-        *,
-        inplace: Literal[True],
-    ) -> None: ...
-    @overload
     def fillna(
         self,
         value,
         method: str | None = ...,
         axis: Axis = ...,
         limit: int | None = ...,
         downcast: dict | None = ...,
-        *,
-        inplace: Literal[False],
+        inplace: Literal[False] = ...,
```

Comment:
Need to preserve the ordering of the params.  `inplace` comes before `limit`,  then `downcast`